### PR TITLE
Amq current state API call fix 

### PR DESCRIPTION
**Pull request by aneeshkp**
Event Pull Status Notification GET Method not working as per o-Ran spec

### CODE REVIEW

File: examples/manifests/kustomization.yaml [hunk 1]
```diff
@@ -14,5 +14,5 @@ images:
   newName: quay.io/redhat-cne/cloud-event-consumer
   newTag: latest
 - name: cloud-event-sidecar
-  newName: quay.io/redhat-cne/cloud-event-proxy
+  newName: quay.io/aneeshkp/origin-cloud-event-proxy
```
Comment (nishant-parekh) [inline examples/manifests/kustomization.yaml:17]: I think this needs to point to the redhat-cne version

**Review by nishant-parekh — COMMENTED** [review]
small update but otherwise looks good

File: examples/manifests/kustomization.yaml (hunk 1 above)
Comment (aneeshkp) [reply examples/manifests/kustomization.yaml]: Done

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**